java/java.lsp.server/vscode/src/extension.ts

```diff
@@ -896,6 +910,11 @@ function activateWithJDK(specifiedJDK: string | null, context: ExtensionContext,
     }
 }
 
+function openSSHSession(username: string, host: string) {
+    const terminal = vscode.window.createTerminal(`SSH: ${username}@${host}`);
```

Comment:
could be the Compute Instance node's name/display name be somehow incoporated ? The user would easily connect the SSH session to the cloud browser contents.